pkg/bindings/kube/kube.go

```diff
@@ -0,0 +1,97 @@
+package kube
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/containers/image/v5/types"
+	"github.com/containers/podman/v4/pkg/auth"
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/sirupsen/logrus"
+)
+
+func Play(ctx context.Context, path string, options *PlayOptions) (*entities.PlayKubeReport, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	return PlayWithBody(ctx, f, options)
+}
+
+func PlayWithBody(ctx context.Context, body io.Reader, options *PlayOptions) (*entities.PlayKubeReport, error) {
+	var report entities.PlayKubeReport
+	if options == nil {
+		options = new(PlayOptions)
+	}
+
+	conn, err := bindings.GetClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	params, err := options.ToParams()
+	if err != nil {
+		return nil, err
+	}
+	if options.SkipTLSVerify != nil {
+		params.Set("tlsVerify", strconv.FormatBool(options.GetSkipTLSVerify()))
+	}
+	if options.Start != nil {
+		params.Set("start", strconv.FormatBool(options.GetStart()))
+	}
+
+	header, err := auth.MakeXRegistryAuthHeader(&types.SystemContext{AuthFilePath: options.GetAuthfile()}, options.GetUsername(), options.GetPassword())
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := conn.DoRequest(ctx, body, http.MethodPost, "/play/kube", params, header)
```

Comment:
```suggestion
	response, err := conn.DoRequest(ctx, body, http.MethodPost, "/play/kube", params, header)
```

Since we are adding new bindings, we should also add a new endpoint here. `/play/kube` -> `/kube/play`

Have a look at `pkg/api/server/registry_play.go` and other files on how to do that.